docs/contributing.md

```diff
@@ -160,3 +171,15 @@ When a commit with the `feat` or `fix` [commit type prefix](#commit-type-prefixe
 If the PR is left alone, it will continue to be updated with new releases as more commits appear on the `main` branch.
 
 Before approving and merging the release PR, make sure you review it. You need to check the package versions that it updates to make sure you’re only releasing the things you expect.
+
+### Correcting releases
+
+Sometimes the releases that Release Please decides to create may be incorrect, because of the way it bumps packages for _all_ changed files in a PR. Considering [pull request scope](#pull-request-scope) is important, but sometimes it's unavoidable. In this case it's possible to change the releases that an already-merged PR will create.
+
+[Update the PR description with a special override to correct the release type](https://github.com/googleapis/release-please/blob/main/README.md#how-can-i-fix-release-notes) and re-run the Release Please command locally (where `XXXXXX` is a GitHub token with write access to the Reliability Kit repo):
```

Comment:
Is it worth adding instructions on how to create such a token?

I'm immediately wondering if it would be okay to create such a token using my own GitHub account or if it should be generated using a team/group bot account.

Reply:
Good point 👍 is it clearer now?